frame/contracts/proc-macro/src/lib.rs

```diff
@@ -617,6 +658,7 @@ fn expand_functions(def: &EnvDef, expand_blocks: bool, host_state: TokenStream2)
 					.memory()
 					.expect("Memory must be set when setting up host data; qed")
 					.data_and_store_mut(&mut __caller__);
+				#body_strace
```

Comment:
we care about the result as well, will move that after the call and log that as well